docs/content/using-tesseracts/advanced.md

```diff
@@ -38,3 +38,34 @@ $ tesseract run --gpus 0 --gpus 1 helloworld apply '{"inputs": {"name": "Osborne
 ```
 
 The GPUs are indexed starting at zero with the same convention as `nvidia-smi`.
+
+## Debug mode
+
+`tesseract serve` supports a `--debug` flag; this has two effects:
+  *  Tracebacks from execution are returned in the response body, instead of a generic 500 error.
+     This is useful for debugging and testing, but unsafe for production environments.
+  *  Aside from listening to the usual Tesseract requests, a debugpy server is also started in
+     the container, and the port it's listening to is forwarded to some free port on the host which
+     is displayed in the cli when spinning up a tesseract via `tesseract serve`. This allows you to perform
+     remote debugging sessions.
+
+In particular, if you are using vscode , here is a sample launch config to attach to a running Tesseract in
+debug mode:
```

Comment:
If you're including the launch config we also need to state what to do with it (or give a link to VSCode tutorials / docs that cover this).